src/main/resources/archetype-resources/silo/src/main/resources/application-testing.yml

```diff
@@ -2,5 +2,9 @@ spring:
     main:
         show_banner: false
 
+    jpa:
+         hibernate:
+           ddlAuto: none
```

Comment:
@bjoernhaeuser I'm not sure I get it. How do you prevent Hibernate from creating tables from within `BaseIntegrationTest`? I understand that even before the first test is run, this property kicks-in, creates the tables and afterwards drops the DB. In fact, we had an issue with this property in employee-tracking-silo where integration tests behaviour changed a lot after adding this property.